.circleci/config.yml

```diff
@@ -17,3 +17,5 @@ jobs:
       - run: sudo apt install python3 -y
       - run: python3 ./scripts/license_check.py
       - run: go test -v -race $(go list ./src/utils | grep -v vendor)
+      - run: go test ./src/container
+      - run: go test ./src/utils
```

Comment:
Maybe replace these three lines with:
```
go test -v -race ./...
```
It will run all test files in the codebase.

Reply:
Will do

Reply:
On second thought, not sure if `-race` should be there considering all of our tests aren't for concurrent functionalities.

Reply:
So do I keep them as 3 separate tests?

Reply:
No no, just 
```
go test -v ./...
```
should be fine